cx.ring.Ring.json

```diff
@@ -11,6 +11,7 @@
         "--share=network",
         "--device=all",
         "--share=ipc",
+        "--extension=org.freedesktop.Plaform.VAAPI=version=1.6",
```

Comment:
You don't need this, that extension point comes with the runtime you depend upon